ServerCore/Pages/Pieces/InternMeta.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/InternMeta/{puzzleId}"
+@model ServerCore.Pages.Pieces.InternMetaModel
+
+@{
+    ViewData["Title"] = "Intern Puzzleday Meta";
+}
+
+<h2>Metapuzzle Pieces Earned</h2>
+
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.EarnedPieces[0].Contents)
```

Comment:
Does this cause the page to crash if there are no pieces?

Reply:
No - all our index pages use the same approach and do not explode...